libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -37,27 +37,17 @@ public static void setConfigService(ConfigService configService) {
         VcHelper.configService = configService;
```

Comment:
This feels like a pretty dodgy pattern - having a static class that requires you to call a method on it before using, it should probably just be a non-static class (or this method could be absorbed into another service).

I'm not sure it's worth changing at this stage - we may be revising the concept of successful VCs entirely anyway.